exercises/custom-set/custom_set_test.go

```diff
@@ -194,22 +131,25 @@ func TestHas(t *testing.T) {
 	}
 }
 
-func TestIsEmpty(t *testing.T) {
-	for _, tc := range emptyCases {
+func TestAdd(t *testing.T) {
+	for _, tc := range addCases {
 		s := NewFromSlice(tc.set)
-		got := s.IsEmpty()
-		if got != tc.want {
-			t.Fatalf("%v IsEmpty = %t, want %t", s, got, tc.want)
+		s.Add(tc.ele)
+		want := NewFromSlice(tc.want)
+		if !Equal(s, want) {
```

Comment:
ah, yes, I remember what I did now, I had them all in the order suggested, and then forgot while cleaning things up a bit and grouped all the methods together, silly oversight on my part, thanks again for the catch @petertseng